src/sql-parser/src/parser.rs

```diff
@@ -1654,7 +1655,66 @@ impl<'a> Parser<'a> {
             vec![]
         };
 
-        Ok(CsrConnector {
+        Ok(CsrConnectorAvro {
+            url,
+            seed,
+            with_options,
+        })
+    }
+
+    fn parse_csr_connector_proto(&mut self) -> Result<CsrConnectorProto<Raw>, ParserError> {
+        let url = self.parse_literal_string()?;
+
+        let seed = if self.parse_keyword(SEED) {
+            if self.parse_keyword(COMPILED) {
+                let key = if self.parse_keyword(KEY) {
+                    self.expect_keyword(SCHEMA)?;
+                    let message_name = self.parse_literal_string()?;
+                    let schema = self.parse_byte_string()?;
```

Comment:
Super minor, since the syntax here won't be user visible, but the SQL-y way to do this syntax would be:

```
KEY SEED COMPILED SCHEMA '\xabcdef....' MESSAGE '.Foo'
```

(I.e., stick the word `MESSAGE` between the compiled schema and the message name.)